src/SearchField/SearchField.scss

```diff
@@ -1,14 +1,31 @@
+@import "variables";
+
 .pgn__searchfield {
-    @extend .border;
-    @extend .rounded;
     transition: $input-transition;
+    border: $search-border-width solid $search-border-color;
+
+    &:disabled {
+        opacity: $search-disabled-opacity;
+        pointer-events: none;
+    }
 
     &.has-focus {
-        box-shadow: $input-focus-box-shadow;
+        box-shadow: 0 0 0 $search-border-width $search-border-color-interaction;
+
+        .pgn__searchfield_wrapper {
+            box-shadow: 0 0 0 $search-border-width $search-border-color-interaction;
+        }
+    }
+
+    &:hover,
+    &:active {
+        border-color: $search-border-color-interaction;
     }
 
     .form-control {
         @extend .border-0;
```

Comment:
I believe it's an anti-pattern to use `@extend` for Paragon styles because the extended rules include a `!important` which makes it hard to theme the components.

As part of this PR, let's remove the `@extend` throughout this file and use SCSS variables or vanilla CSS instead (e.g., `border: none;` here).

Reply:
Agree, updating this